modules/local/igv.nf

```diff
@@ -42,7 +32,7 @@ process IGV {
     igv_files_to_session.py igv_session.xml igv_files.txt ../../genome/${fasta.getName()} --path_prefix '../../'
 
     cat <<-END_VERSIONS > versions.yml
-    ${getProcessName(task.process)}:
+    "${task.process}":
```

Comment:
What about the commented out lines at the bottom of this script? https://github.com/JoseEspinosa/nf-core-chipseq/blob/865210afa074301fad8e83429dbae93b762ea0ae/modules/local/igv.nf#L40-L41

Reply:
TBH, they were already there before I started reimplementing the pipeline and thought that were lines that at some point will be reused again